package.json

```
@@ -1,11 +1,11 @@
 {
-	"name": "@govtechsg/singpass-myinfo-oidc-helper",
-	"version": "7.3.0",
+	"name": "@choonkeat/singpass-myinfo-oidc-helper",
+	"version": "7.3.1",
```

well, maybe to merge, switch this back

```suggestion
	"name": "@govtechsg/singpass-myinfo-oidc-helper",
	"version": "7.3.1",
```